templates/ifcfg-vxlan.j2

```diff
@@ -0,0 +1,11 @@
+DEVICE={{ item.device }}
+TYPE=VXLAN
+VNI={{ item.vni | default(vxlan_vni) }}
+GROUP={{ item.group }}
+TTL={{ item.ttl | default(vxlan_ttl) }}
+PHYS_DEV={{ item.phys_dev | default(vxlan_phys_dev) }}
+DSTPORT={{ item.dstport | default(vxlan_dstport) }}
+IPADDR={{ item.ipaddr }}
+PREFIX={{ item.prefix }}
```

Comment:
You won't want to assign these in DHCP mode, or even sometimes in static config mode. Suggest making conditional on `item.ipaddr` being defined.